### PR TITLE
Murad      Week 2 + Week 3

### PR DESCRIPTION
**Pull request by g6w9a**

### CODE REVIEW

File: Week2/Murads_homework/scripts/5.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+function vijf(){
+    console.clear();
+
+    var z = 7.25; //5.1
+    console.log(z) //5.2
+    var a = Math.round(z); //5.3
+    console.log(a) //5.4
+    var highest = Math.max(z, a); //5.5
```
Comment (marcianoviereck) [inline Week2/Murads_homework/scripts/5.js:8]: Nice solution!

File: Week2/Murads_homework/scripts/8.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    console.log("The value of my variable datas is: " + datas);
+    console.log("The type of my variable datas will be: object");
+    console.log(typeof(datas));
+
+    let dict = {"num":num, "name":name, "bool":bool, "list":lst, "datas":datas};
+   
+    for (const [key, value] of Object.entries(dict)){
+        item = {'k':key, 'v':value};
+        delete dict[key];
+        for (var data in dict){
```
Comment (marcianoviereck) [inline Week2/Murads_homework/scripts/8.js:34]: wow nice solution!

File: Week2/Murads_homework/scripts/9.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+    let x = 7;
+    x = x % 3;
+    console.log("7 mod 3 = " + x); 
+
+    let y = 15;
+    console.log("15 mod 5 = " + (y%5)); 
+
+    let z = 19;
+    console.log("19 mod 7 = " + (z%7));
```
Comment (marcianoviereck) [inline Week2/Murads_homework/scripts/9.js:12]: Week 2 homework approved

File: Week3/scripts/3.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    vehicleType("blue", 2);
+
+    console.log((3===3) ? "Yes" : "No");
+
+    function vehicle(col, code, age){
+        if(code != 1 && code !=2){
+            console.log("Please enter a valid code number!");
+            return;
+        }
+        console.log("a " + col + ((age<3)? " new":" used") + ((code===1)? " car": " motorbike"));
```
Comment (marcianoviereck) [inline Week3/scripts/3.js:42]: Ah, compact solution! It's not easy to read, so you can make it more readable if you want, but the solution itself is fine

File: Week3/scripts/3.js [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+    let teachers = {HtmlCss:['Rob','Philip'], Git:['Unmesh', 'Hardit', 'Bonan'], JS:['Maher','Sander']}
+    obj['teachers'] = teachers;
+    obj['languages'] = ['HTML/CSS', 'BASH', 'JAVASCRIPT'];
+
+    console.log(obj);
+    
+
+    console.log("'==' just checks the items but it doesn't check the types of items. But with '===' we can also check the types.");
+    console.log("for the objects it also checks the adresses in disk. (My assumption)");
```
Comment (marcianoviereck) [inline Week3/scripts/3.js:92]: Yep, [here](http://adripofjavascript.com/blog/drips/object-equality-in-javascript.html) you can read more about it!

**Review by marcianoviereck — APPROVED** [review]
Nicely done, the homework looks good! I have some small suggestions here and there, but the homework is approved anyways

**Comment by wouterkleijn** [comment]
Great job @Hdvndgr Thanks for submitting the homework. Thanks also @marcianoviereck for providing feedback.

**Pull request closed without merging**